accounts/views.py

```diff
@@ -82,6 +90,34 @@ def login_user(request):
     return render(request, template, context)
 
 
+@api_view(("POST",))
+@csrf_exempt
```

Comment:
@autn I'm curious as to why we need to exempt CSRF here. Is it just for the login method that we don't need CSRF and other methods do need CSRF?

Where I'm confused also is that for instance this answer says that anonymous requests do not require CSRF: https://stackoverflow.com/a/30875830/2532070

Reply:
@YPCrumble from the answer:
```If you're using an AJAX style API with SessionAuthentication, you'll need to include a valid CSRF token for any "unsafe" HTTP method calls, such as PUT, PATCH, POST or DELETE requests.```
Because we are using  POST method, so we need to exempt CSRF. Also, it is just a single method, not a DRF object.